### PR TITLE
Use the local ORCID icon

### PR DESCRIPTION
**Pull request by egonw**

### CODE REVIEW

File: scholia/app/templates/base.html [hunk 1]
```diff
@@ -35,9 +35,9 @@
 
       try {
           var orcid = item.claims.P496[0].mainsnak.datavalue.value;
-          $( '#details' ).append( '<div><!-- a href="https://orcid.org/"><!-- img alt="' +
-	      'ORCID logo" src="https://orcid.org/sites/default/files/images/orcid_16x16.png"' +
-	      'width="16" height="16" hspace="4" /></a --> <a href="https://orcid.org/' +
+          $( '#details' ).append( '<div><a href="https://orcid.org/"><img alt="' +
+	      'ORCID logo" src="/static/images/orcid_16x16.gif"' +
```
Comment (fnielsen) [inline scholia/app/templates/base.html]: I think there is a problem with "/..." due to tools.wmflabs.org having the /scholia prefix.

Comment (egonw) [reply scholia/app/templates/base.html]: I copied that from another file, just to prevent issues like that :(

Comment (egonw) [reply scholia/app/templates/base.html]: I'll push a fix...

Comment (egonw) [reply scholia/app/templates/base.html]: testing the fix...

**Comment by egonw** [comment]
@fnielsen, OK, better /static/ solution pushed... (as third commit to this PR)

**Pull request merged**